wiki/Mappers_Guild/tr.md

```diff
@@ -0,0 +1,62 @@
+
+Loncanın organizasyonu ve tartışması [osu!dev Discord sunucusundaki](https://discord.gg/ppy) `#mappers-guild` kanalında bulunabilir.
+
+## Sistem
+
+*Mappers' Guild sistemi hakkında güncel bilgiye erişmek için [loncanın web sayfasındaki SSS bölümüne](https://mappersguild.com/faq) bakınız.*
+
+3 ya da daha fazla dereceli mape sahip mapperlar loncaya istedikleri zaman [Mappers' Guild web sayfası](https://mappersguild.com/) üzerinden osu! hesaplarını doğrulatmak suretiyle katılabilirler. Diğer herkes ise web sayfasının görevlerini ve mapleme aktivitesini gözlemlemekte serbesttir.
+
+Mapperlar Mappers' Guild'e bağlı olan featured artistlerin maplerinin derecelendirilme prosedürüne katkıda bulunarak ödüller kazanabilir. Bu, temel olarak beatmap oluşturmaya odaklanır, ancak [modlama](/wiki/Modding) yaparak, [storyboard](/wiki/Storyboards) oluşturarak, ya da lonca içindeki görevleri tamamlayarak katkıda bulunan kullanıcılara da ödül verilir.
+
+### Partiler
+
+Partiler birlikte görev tamamlamak için kayıt yaptıran mapper gruplarıdır. Mapperlar tek seferde istedikleri sayıda partiye katılabilirler.
+
+Özel görevlere katılmak üst lonca rütbesi gerektirir. Bir partinin üst bir lonca rütbesinden sayılması için, üyelerinin en az yarısının gerekli rütbe eşiti ya da üstü olması gereklidir.
+
+### Görevler
+
+Görevler, mapperlar featured artist mapleri oluştururken belirli gereksinimleri sağladıklarında onları ödüllendirir. Görevler genellikle yeni bir featured artist için belirli bir sayıda beatmap oluşturmayı kapsar, ancak bazı görevler ufak numaralar gerektirir.
+
+Her bir görevin parti büyüklüğü/rütbesi şartları, kabul etme bedeli, ve bitiş tarihi vardır. Bir partinin bütün üyeleri görevi kabul etmek için belirtilen bedeli ödemek zorundadır. Bu bedel genellikle tek kişilik görevler ve numaralar gerektiren görevler için daha büyüktür.
+
+Görevlerin süresi 3 aylık inaktiflik sonrasında dolar. Herhangi bir kullanıcı mevcut Mappers' Guild puan bakiyesinden harcayarak görevleri geri açabilir.
+
+Mappers' Guild için görevlere katılım zorunlu değildir. Eğer isterse bir kullanıcı tek başına beatmapler oluşturarak ödüller kazanabilir.
+
+### Ödüller
+
+Ödüller bir puan sistemi aracılığı ile dağıtılır. Bir kullanıcı belirli bir puan eşiğine vardığında, bir profil rozeti ile belirtilen bir rütbe elde ederler. Sırasıyla 100, 250 ve 500 puanda olmak üzere üç farklı profil rozeti seviyesi vardır.
+
+Ödüller kullanıcının toplam puanlarına dayalıdır. Bir kullanıcının mevcut puanları osu!'nun [kudosu](/wiki/Modding/Kudosu) "toplam kazanılan" ve "mevcut" puan havuzları ile aynı modeli takip eden onun toplam puanlarının bir alt sekmesidir.
```

Comment:
```suggestion
Ödüller kullanıcının toplam puanına dayalıdır. Bir kullanıcının mevcut puanı ise, osu!'nun [kudosu](/wiki/Modding/Kudosu) "toplam kazanılan" ve "mevcut" puan havuzları gibi, toplam puanlarının bir kısmıdır.
```
`subsection` is hard to translate. maybe we should do a bigger change here

Reply:
```suggestion
Ödüller kullanıcının toplam puanına dayalıdır. Bir kullanıcının mevcut puanı ise, osu!'nun [kudosu](/wiki/Modding/Kudosu) "toplam kazanılan" ve "mevcut" puan havuzları gibi, toplam puanlarının bir yan havuzudur.
```
this should do I think